okta-hosted-login/okta-aspnet-mvc-example/okta-aspnet-mvc-example.csproj

```diff
@@ -258,16 +284,7 @@
     <VisualStudio>
       <FlavorProperties GUID="{349c5851-65df-11da-9384-00065b846f21}">
         <WebProjectProperties>
-          <UseIIS>True</UseIIS>
-          <AutoAssignPort>True</AutoAssignPort>
-          <DevelopmentServerPort>63475</DevelopmentServerPort>
-          <DevelopmentServerVPath>/</DevelopmentServerVPath>
-          <IISUrl>https://localhost:44314/</IISUrl>
```

Comment:
Did you mean removing these config lines?

Reply:
Yes, these settings caused conflicts with other samples which lead to errors in IIS Express.  Removing them allows the settings to be unique per user without conflicts and errors.